problems/[230605] Sort Colors.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/sort-colors
+ *
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+const sortColors = function (nums) {
+  for (let n = 0; n < nums.length - 1; n++) {
+    for (let i = 0; i < nums.length - 1; i++) {
+      if (nums[i] > nums[i + 1]) {
+        const a = nums[i + 1];
+        nums[i + 1] = nums[i];
+        nums[i] = a;
```

Comment:
```suggestion
        [nums[i], nums[i + 1]] = [nums[i + 1], nums[i]];
```
배열 구조분해할당을 이용해서 swaping을 할 수도 있을 것 같습니다.